qdrant-landing/content/documentation/frameworks/testcontainers.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Testcontainers
+weight: 2700
+---
+
+# Testcontainers
+
+[Testcontainers](https://testcontainers.com/) is an open source framework for providing throwaway, lightweight instances of databases, message brokers, web browsers, or just about anything that can run in a Docker container.
+
+Qdrant is available as a [Testcontainers module](https://testcontainers.com/modules/qdrant/) in multiple languages facilitating the spawning of a Qdrant instance for end-to-end testing.
```

Comment:
Since you're quoting directly from the Testcontainers website:

```suggestion
Qdrant is available as a [Testcontainers module](https://testcontainers.com/modules/qdrant/) in multiple languages. It  facilitates the spawning of a Qdrant instance for end-to-end testing.

As noted by [Testcontainers](https://testcontainers.com/), it "is an open source framework for providing throwaway, lightweight instances of databases, message brokers, web browsers, or just about anything that can run in a Docker container."
```